src/scenes/Search/store.js

```diff
@@ -64,17 +77,20 @@ export default class SearchStore {
           throw new Error(`Invalid tab index: ${this.tabIdx}`);
         }
       }
+      this.loading = false;
+      this.loaded = true;
     });
   }
 
-  async fetch(phrase) {
-    let result = [];
-    if (_.isEmpty(phrase)) return result;
-    let oracles = await searchOracles(phrase);
-    oracles = _.uniqBy(oracles, 'txid').map((oracle) => new Oracle(oracle, this.app));
-    let topics = await searchTopics(phrase);
-    topics = _.uniqBy(topics, 'txid').map((topic) => new Topic(topic, this.app));
-    result = [...oracles, ...topics];
-    return _.orderBy(result, ['endTime']);
+  async fetch() {
+    if (_.isEmpty(this.phrase)) return;
+    const orderBy = { field: 'blockNum', direction: SortBy.DESCENDING };
+    let oracles = await searchOracles(this.phrase, null, orderBy);
+    oracles = _.uniqBy(oracles, 'txid');
```

Comment:
is this necessary to do the `uniqBy`? IIRC, `txid` should be unique among all Oracles.

Reply:
I was referring to `scenes/Activities/Finalize`, `scenes/Activities/ResultSetting` and `scenes/Activities/Withdraw`, all these store files do this and I thought it was a safety action.

But yeah, the results show the same if remove it.

Reply:
Ok, I will remove these in my test branch